core/src/main/scalajvm/sttp/client4/httpclient/HttpClientBackend.scala

```diff
@@ -103,8 +127,46 @@ abstract class HttpClientBackend[F[_], S, P, B](
     monad.map(body)(Response(_, code, "", headers, Nil, request.onlyMetadata))
   }
 
+  protected def createSimpleQueue[T]: F[SimpleQueue[F, T]]
```

Comment:
do these methods need to be in the super-class? are they used here at all?

Reply:
this is needed in WebSocket implementation and used by all classes in `httpclient` package

Reply:
yes, but does it need to be declared in `HttpClientBackend`? Is there any common code that uses this method?

Reply:
It was declared in `HttpClientAsyncBackend` before because it was only needed in async implementation (`HttpClientFutureBackend`). I thought it would be a good idea to move it to `HttpClientBackend` since it is used in `HttpClientSyncBackend` which implements `HttpClientBackend` and `HttpClientFutureBackend` which implements `HttpClientAsyncBackend` which implements `HttpClientBackend`.
So from my point of view it is not a bad idea to have it here. WDYT?

Reply:
from the point of view of the `HttpCLientBackend` class, this method is not used, and hence an implement detail of child classes - so it shouldn't be declared there

Reply:
I can see your point and it makes sense to me right now. Thanks for the explanation :)